Add information about Alternate Content Sources (#1916)
theforeman/foreman-documentation: guides/common/modules/con_alternate-content-sources.adoc

> [id="Managing_Alternate_Content_Sources_{context}"]

Reviewer: File name does not match your title/anchor.

= Managing Alternate Content Sources

Alternate content sources define alternate paths to download content during synchronization.

> The content itself is downloaded from the alternate content source, and only the metadata is downloaded from the upstream URL.

Reviewer: ```suggestion
The content itself is downloaded from the alternate content source and only the metadata is downloaded from the upstream URL.
```

Alternate content source can be used to speed up synchronization if the content is located on the local filesystem or on a nearby network.

> You can set up alternate content sources for {Project}, as well as for {SmartProxy}.

Reviewer: ```suggestion
You can set up alternate content sources for {Project} and {SmartProxies}.
```
